### DISCUSSION TITLE
Ability to set volume increments

### DISCUSSION THREAD

**Discussion by neauxm** [MonitorControl/MonitorControl#1436] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

Currently the volume changes in increments of 6 (0,6,12,18,24,30), when using keyboard shortcuts or mouse shortcuts.

### Describe the solution you'd like

 Can we have control over the increment levels? I find it too loud or too quite 

### Describe alternatives you've considered

Using the volume slider from the icon quick actions

### Anything else?

NA

**Comment by waydabber** [comment]
Hi @neauxm, you can redefine the DDC volume range already in the app - that will proportionately change the increments as well.
